<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckPriceDialog</class>
 <widget class="QDialog" name="CheckPriceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Check Price</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="lineBarcode">
     <property name="placeholderText">
      <string>Search barcode</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelName">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="LibGUI::TableWidget" name="tableWidget" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LibGUI::TableWidget</class>
   <extends>QWidget</extends>
   <header location="global">tablewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
